runtimes/alibi-detect/mlserver_alibi_detect/protocols/seldon_http.py

```diff
@@ -0,0 +1,64 @@
+from .request_handler import RequestHandler
+from mlserver.errors import InferenceError
+import numpy as np
+from enum import Enum
+
+
+class SeldonPayload(Enum):
+    TENSOR = 1
+    NDARRAY = 2
+    TFTENSOR = 3
+
+
+def _extract_list(body: dict) -> np.array:
```

Comment:
I wonder if these seldon v1 protocol extraction utils can be made available upstraem, as the `if statement` extracting from data / ndarray /tfttensor is probably relevant in tempo, req logger, etc (definitely out of this PR but perhaps could be part of the exploration of the Handler classes

Reply:
Yeah good point. we also discussed this. I think a good idea that Adrian mentioned was to have them as codecs in MLserver that can be reused across runtimes and projects